src/components/fields/image-upload/image-review/image-review.tsx

```diff
@@ -199,24 +216,37 @@ export const ImageReview = (props: IProps) => {
 		setActivePrompt(null);
 	};
 
+	const handleRemovePendimgImage = () => {
```

Comment:
```suggestion
	const handleRemovePendingImages = () => {
```

Reply:
removed this function as the fn to dismissreviewmodal is small enough to put this in